server/build.gradle

```diff
@@ -60,6 +60,12 @@ dependencies {
     implementation "com.azure:azure-storage-blob:${versions.azure}"
     implementation "io.springfox:springfox-boot-starter:${versions.springfox}"
     implementation "com.google.guava:guava:${versions.guava}"
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.12.5'
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.12.5'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.5'
+    implementation 'com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.12.5'
+    implementation 'com.fasterxml.woodstox:woodstox-core:6.2.4'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.12.5'
```

Comment:
Jackson is already included with `spring-boot-starter-json`. Is it really necessary to declare the dependency here? If yes, please extract the common version to the `versions` map.

Reply:
yeah but I need a newer version as the included one is old and has a bug when parsing arrays in xml, is there a better way to ask for a newer version?

Reply:
I don't know. Let's keep this in mind for later.